src/content/extensions/multipleroutes/src/options.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+// Saves options.
+function saveOptions() {
+  var multiRoutes;
+  var nonProxiedUdp;
+  var radios = document.getElementsByName('routeselection');
+  for (var i = 0, length = radios.length; i < length; i++) {
+    if (radios[i].checked) {
+      multiRoutes = i < 1;
+      nonProxiedUdp = i < 2;
+      break;
+    }
+  }
+
+  chrome.privacy.network.webRTCMultipleRoutesEnabled.set({
+    'value': multiRoutes
+  });
+  try {
+    chrome.privacy.network.webRTCNonProxiedUdpEnabled.set({
+      'value': nonProxiedUdp
+    });
+  } catch (err) {
+    // Shouldn't run into here.
+  }
+}
+
+function restoreRadios(multiRoutes, nonProxiedUdp) {
+  var radios = document.getElementsByName('routeselection');
+  if (multiRoutes) {
+    if (nonProxiedUdp) {
+      radios[0].checked = true;
+    } else {
+      radios[0].checked = false;
+      alert(
```

Comment:
If this happens, the customer has gone through the pain to modify preferences file. I think we should just leave as it is.